### PR TITLE
Updated redirects and readme for gerbil

### PR DESCRIPTION
**Pull request by MichaelRoeder**

### CODE REVIEW

File: gerbil/README.md [hunk 1]
```diff
@@ -1,28 +1,46 @@
 # GERBIL
-GERBIL is a general entity annotator benchmark, GERBIL QA is a benchmarking framework for question answering systems and GERBIL SW is a benchmark framework for other semantic web systems such as attribute prediction. This redirect links to the instances of GERBIL, GERBIL QA, GERBIL SW and their SPARQL endpoints hosted by AKSW.
+* GERBIL is a general entity annotator benchmark,
+* GERBIL QA is a benchmarking framework for question answering systems, and
+* GERBIL KBC is a benchmark framework for tasks related to knowledge graph curation.
+
+This redirects the following links to the instances of GERBIL, GERBIL QA, GERBIL KBC and their SPARQL endpoints hosted by Paderborn University:
```
Comment (TallTed) [inline gerbil/README.md:6]:
```suggestion
This redirects the following links to the instances of GERBIL, GERBIL QA, GERBIL KBC, and their SPARQL endpoints hosted by Paderborn University:
```

**Comment by dgarijo** [comment]
@MichaelRoeder,
can you please state who the maintainer for this w3id is? Also please double check the quotes in the htaccess, I am not sure it'll work like that

**Comment by MichaelRoeder** [comment]
@dgarijo Thanks for your feedback :smiley: 

## Maintainer
The project is hosted at the moment by the DICE research group (dice-research.org) at Paderborn University. I am the main maintainer and created the first version of the redirect 6 years ago (https://github.com/perma-id/w3id.org/commit/01ff2b4bfbac5eb55b06bcc7d7435bd3a1fd81d5)

Should I add more information to the README file with respect to that?

## Quotes

Which quotes are we talking about? Can you please give me the exact line? (Or do you mean all the lines)
Actually all the lines worked before. I only changed the target address.

**Comment by MichaelRoeder** [comment]
I added a list of maintainers to the README file.

**Pull request merged**